Framework tests not enough args in call

```
# github.com/kube-reporting/metering-operator/test/deployframework [github.com/kube-reporting/metering-operator/test/deployframework.test]
test/deployframework/framework_test.go:22:34: not enough arguments in call to df.NewDeployerConfig
	have (string, string, string, string, string, "github.com/kube-reporting/metering-operator/pkg/apis/metering/v1".MeteringConfigSpec)
	want (string, string, string, string, string, string, string, string, "github.com/kube-reporting/metering-operator/pkg/apis/metering/v1".MeteringConfigSpec)
```

Looks like we are missing information for `catalogSourceName, catalogSourceNamespace, subscriptionChannel`

affected line (framework_test.go):
```go
cfg, err := df.NewDeployerConfig(testNamespace, testMeteringOpRepo, testMeteringOpTag, testReportingOpRepo, testReportingOpTag, spec)
```

constructor (framework.go):
```go
// NewDeployerConfig handles the process of validating inputs before returning
// an initialized Deploy.Config object, or an error if there is any.
func (df *DeployFramework) NewDeployerConfig(
	namespace,
	meteringOperatorImageRepo,
	meteringOperatorImageTag,
	reportingOperatorImageRepo,
	reportingOperatorImageTag,
	catalogSourceName,
	catalogSourceNamespace,
	subscriptionChannel string,
	spec metering.MeteringConfigSpec,
) (*deploy.Config, error)
```

--- a/Makefile
+++ b/Makefile
@@ -126,7 +126,7 @@ e2e: $(DEPLOY_METERING_BIN_OUT)
 	hack/e2e.sh
 
 e2e-upgrade: $(DEPLOY_METERING_BIN_OUT)
-	EXTRA_TEST_FLAGS="-run TestMeteringUpgrades" ./hack/e2e.sh
+	$(MAKE) e2e EXTRA_TEST_FLAGS="-run TestMeteringUpgrades"
 
 e2e-local: reporting-operator-local metering-ansible-operator-docker-build
 	$(MAKE) e2e METERING_RUN_TESTS_LOCALLY=true METERING_OPERATOR_IMAGE_REPO=$(METERING_OPERATOR_IMAGE_REPO) METERING_OPERATOR_IMAGE_TAG=$(METERING_OPERATOR_IMAGE_TAG)

--- a/hack/e2e.sh
+++ b/hack/e2e.sh
@@ -109,6 +109,12 @@ go test \
     ${EXTRA_TEST_FLAGS} \
     2>&1 | tee "$TEST_LOG_FILE_PATH" ; TEST_EXIT_CODE=${PIPESTATUS[0]}
 
+go test \
+    -timeout 30s \
+    -test.v \
+    "./test/deployframework" \
+    2>&1 | tee "$TEST_LOG_FILE_PATH" ; TEST_EXIT_CODE=${PIPESTATUS[0]}
+
 # if go-junit-report is installed, create a junit report also
 if command -v go-junit-report >/dev/null 2>&1; then
     go-junit-report < "$TEST_LOG_FILE_PATH" > "${TEST_JUNIT_REPORT_FILE_PATH}"

